chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html

```diff
@@ -2,7 +2,7 @@
   <h1><b>How to Generate Picks:</b></h1>
 </div>
 <div class="how-to-generate">
-  <img src="/assets/images/Info page -12.png" alt="Generate" style="width:700px" align="center">
+  <img src="src/app/assets/images/Info page -12.png" alt="Generate" style="width:700px" align="center">
 
-  <img src="/assets/images/Info page -13.png" alt="Generate" style="width:700px" align="center">
+  <img src="src/app/assets/images/Info page -13.png" alt="Generate" style="width:700px" align="center">
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted back to assets/images.

Reply:
reverted back to assets/images.